config/environment.js

```diff
@@ -1,5 +1,13 @@
 'use strict';
 
-module.exports = function(/* environment, appConfig */) {
-  return { };
+/**
+ * Export the default config for ember-devtools. By default, enable only
+ * in development.
+ */
+module.exports = function(environment, appConfig) {
+  appConfig['ember-devtools'] = appConfig['ember-devtools'] || {};
+  appConfig['ember-devtools'].enabled = appConfig['ember-devtools'].enabled || environment === 'development';
+  appConfig['ember-devtools'].global = appConfig['ember-devtools'].global || false;
```

Comment:
True, object-assign should do the trick.

On Tue, 9 Jun 2015 00:30 Aaron Ortbals <notifications@github.com> wrote:

> In config/environment.js
> <https://github.com/aexmachina/ember-devtools/pull/10#discussion_r31919594>
> :
>
> > @@ -1,5 +1,13 @@
> >  'use strict';
> >
> > -module.exports = function(/* environment, appConfig */) {
> > -  return { };
> > +/**
> > + * Export the default config for ember-devtools. By default, enable only
> > + * in development.
> > + */
> > +module.exports = function(environment, appConfig) {
> > +  appConfig['ember-devtools'] = appConfig['ember-devtools'] || {};
> > +  appConfig['ember-devtools'].enabled = appConfig['ember-devtools'].enabled || environment === 'development';
> > +  appConfig['ember-devtools'].global = appConfig['ember-devtools'].global || false;
>
> I completely agree, however this is in a node environment and I don't
> believe we can use Ember here. The alternative would be to include
> something like lodash/object/merge
> <https://github.com/ember-cli/ember-cli/blob/7af15c5f5e3ddbf8cdbde84d5b968a053c8c322d/lib/models/project.js#L14>
> or vendor
> https://github.com/emberjs/ember.js/blob/v1.12.0/packages/ember-metal/lib/merge.js
> .
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/aexmachina/ember-devtools/pull/10/files#r31919594>.
>